src/cdk/overlay/overlay-directives.ts

```diff
@@ -272,7 +272,12 @@ export class ConnectedOverlayDirective implements OnDestroy, OnChanges {
 
   /** Builds the overlay config based on the directive's inputs */
   private _buildConfig(): OverlayState {
-    let overlayConfig = new OverlayState();
+    const positionStrategy = this._position = this._createPositionStrategy();
+    const overlayConfig = new OverlayState({
```

Comment:
Ah, gotcha. Instantiating the class lets us apply the default options for things like the direction, backdrop class and scroll strategy.